src/helpers/Blocks.d.ts

```diff
@@ -63,6 +69,9 @@ export interface KernelsEntity {
     linked_kernel: string;
     excess: string;
     excess_sig: ExcessSig;
+    group: string;
+    size: number;
+    color: string;
```

Comment:
I would extend the KernelsEntity with a specific Single Block View (can be addressed in another PR)